data/actions/scripts/other/fluids.lua

```diff
@@ -9,12 +9,22 @@ poison:setParameter(CONDITION_PARAM_STARTVALUE, -5)
 poison:setParameter(CONDITION_PARAM_TICKINTERVAL, 4000)
 poison:setParameter(CONDITION_PARAM_FORCEUPDATE, true)
 
-local fluidType = {3, 4, 5, 7, 10, 11, 13, 15, 19}
-local fluidMessage = {"Aah...", "Urgh!", "Mmmh.", "Aaaah...", "Aaaah...", "Urgh!", "Urgh!", "Aah...", "Urgh!"}
+local fluidMessage = {
+	[3] = "Aah...",
+	[4] = "Urgh!",
+	[5] = "Mmmh.",
+	[7] = "Aaaah...",
+	[10] = "Aaaah...",
+	[11] = "Urgh!",
+	[13] = "Urgh!",
+	[15] = "Aah...",
+	[19] = "Urgh!",
+	[43] = "Aaaah..."
+}
 
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
-	local targetItemType = ItemType(target.itemid)
-	if targetItemType and targetItemType:isFluidContainer() then
+	local targetType = ItemType(target.itemid)
+	if targetType:isFluidContainer() then
```

Comment:
Why did you remove "if targetItem " - what if the target is not an item?

Reply:
Sorry, it was a mistake